components/Forms/AddFeatureForm.tsx

```diff
@@ -0,0 +1,120 @@
+import { Button, FormControl, FormControlLabel, FormHelperText, FormLabel, Input, InputLabel, Radio, RadioGroup, Snackbar, useMediaQuery } from "@mui/material";
+import { Box } from "@mui/system";
+import BodyHeaderContent from "../Home/BodyHeaderContent";
+import { useAddFeatureForm, useEmailForm } from "./CallToActionHook";
+
+const AddFeatureForm: React.FC = () => {
+    const mediumSizeOrHigher = useMediaQuery("(min-width:768px)");
```

Comment:
We should feature and expand on this min-width prop to get the dimensions right across all components on theme level.